### ISSUE TITLE
[bug] Generated links not working in v0.17

### ISSUE THREAD

**Issue by bendoin** [Luzifer/ots#13]
Reproduce (Fedora 31)

```shell
curl -LO https://github.com/Luzifer/ots/releases/download/v0.17.0/ots_linux_amd64.tar.gz

sha256sum ots_linux_amd64.tar.gz
4753078d6aafcdd8cb128217cda2c8f2618ab284b9191718849d7a2bd63ec109  ots_linux_amd64.tar.gz

tar zxf ots_linux_amd64.tar.gz

sha256sum ots_linux_amd64
f2cade9b0fecc0ce468a73de978f16d6ae2f9b9f582b37ac76e50e8a59876b79  ots_linux_amd64

./ots_linux_amd64

curl localhost:3000/api/create -d "secret=thisisarestlinktest"
{"secret_id":"1926fb79-47bb-4c30-ae55-fcc92d38cff2","success":true}

curl localhost:3000/api/read/1926fb79-47bb-4c30-ae55-fcc92d38cff2
{"error":"Secret not found","success":false}
```

Creating link from web ui - http://localhost:3000/#306867d9-7347-475a-acb1-270fffcc8dc2|r6cy4vs1gaf gives message

> This is not the secret you are looking for… - If you expected the secret to be here it might be comprimised as someone else might have opened the link already.

Expected behavior that works with v0.16.1

Web UI - Link works

REST API

```shell
curl localhost:3000/api/create -d "secret=thisisarestlinktestthat will work v0.161"
{"secret_id":"ec82f65a-6a67-494e-b984-44a115352c84","success":true}

curl localhost:3000/api/get/ec82f65a-6a67-494e-b984-44a115352c84
{"secret":"thisisarestlinktestthat will work v0.161","success":true}
```

**Comment by Luzifer** [comment]
Very good catch. The new expiry of secrets in mem-store (#12) did not take into account there might be secrets without expiry date.

This is fixed in `v0.17.1` which is currently building and will be available within the next few minutes.

Thank you for the report! :slightly_smiling_face:

**Comment by Luzifer** [comment]
Release is now available!
